### ISSUE TITLE
Executing model for single dataset

### ISSUE THREAD

**Issue by HrudayR** [radrumond/timehetnet#1]
Firstly, I would like to commend you on your paper. I had a good time reading it and executing the code.
I want to know if it is possible to train the model on a single data set (ETTh) instead of running training on all datasets ?
If possible please let me know how to execute it.

Thank you

**Comment by RafaelDrumond** [comment]
Hi sorry for the late reply. Do you mean sampling tasks from a single data set? It is possible but might require some hacking. Let me try explaining how our code works.

First, you need your data-set preprocessed. Since you mentioned ETTh we can use that one.

go to [splits](https://github.com/radrumond/timehetnet/tree/main/splits)/regensplits.py

This code will create fold splits of train test and validation. if you check the ```splits_new``` on line 67 it defines the paths to each data-set on each fold and each split, so the first index (0-4) represents each of the 5 folds and the second one (0-2) is respectively "train", "validation", and "test".

In other words, you just need to edit this to define which data sets will be in train, validation and test for each of 5-fold.

You can customize this to either add new folds or data sets.

Keep in mind that if you add the same training data-set to validation, and test, there's no guarantee that these will not share the same samples and features as in train.

Does that help? Let me know if you need any more help.

**Comment by RafaelDrumond** [comment]
By the way, make sure to regenerate your test set if you change those.
